### PR TITLE
Admin dashboard

### PR DESCRIPTION
**Pull request by truongnv0211**
Admin dashbroad

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -3,4 +3,23 @@ class ApplicationController < ActionController::Base
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
   include SessionsHelper
+
+  def logged_in_user
```
Comment (phamkykhoi) [inline app/controllers/application_controller.rb]: Insert protected above line

File: app/helpers/admin/dashboard_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module Admin::DashboardHelper
```
Comment (phamkykhoi) [inline app/helpers/admin/dashboard_helper.rb]: Remove this file

File: config/routes.rb [hunk 3]
```diff
@@ -10,4 +11,9 @@
   delete "logout" => "sessions#destroy"
 
   resources :users
+
+  namespace :admin do
+    get "/" => "dashboard#index"
```
Comment (phamkykhoi) [inline config/routes.rb]: Change root   "dashboard#index"

File: db/migrate/20150625031439_add_default_value_for_admin.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+class AddDefaultValueForAdmin < ActiveRecord::Migration
+  def change
+    change_column :users, :admin, :boolean, :default => false
```
Comment (phamkykhoi) [inline db/migrate/20150625031439_add_default_value_for_admin.rb]: not use => 
use :

File: app/controllers/application_controller.rb [hunk 5]
```diff
@@ -3,4 +3,24 @@ class ApplicationController < ActionController::Base
...
+      store_location
+      flash[:danger] = t :required_user_logged
+      redirect_to login_url
+    end
+  end
+
+  def required_admin
+    unless logged_in? && current_user.admin?
+      flash[:danger] = t :required_admin_logged
+      if current_user.nil?
```
Comment (dieunb) [inline app/controllers/application_controller.rb]:
```
+      redirect_to current_user.nil? ? root_path : current_user
```

**Comment by phamkykhoi** [comment]
OK

**Comment by dieunb** [comment]
OK

**Pull request merged**